lingui.config.ts

```diff
@@ -0,0 +1,13 @@
+// import { formatter } from '@lingui/format-json'
```

Comment:
I'd like to move i18n to the separate PR as this isn't related to the main scope

Reply:
we had to upgrade to v4 for vite. so it requires the fixes here.
i'll see if it is possible to still keep lingui v3